org.x.adriconf.json

```diff
@@ -0,0 +1,183 @@
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftp.acc.umu.se/pub/GNOME/sources/atkmm/2.24/atkmm-2.24.2.tar.xz",
+                    "sha256": "ff95385759e2af23828d4056356f25376cfabc41e690ac1df055371537e458bd"
+                }
+            ]
+        },
+
+        {
+            "name": "graphene-gobject-1.0",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://archive.ubuntu.com/ubuntu/pool/main/g/graphene/graphene_1.8.0.orig.tar.gz",
+                    "md5": "d0ee65f4bdd30a5db4fd8d2ac46cd8cc"
+                }
+            ]
+        },
+
+        {
+            "name": "gtk+-4.0",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/gtk+4.0/3.90.0-0ubuntu2/gtk+4.0_3.90.0.orig.tar.xz",
+                    "sha256": "7bb33b4101a4b3e51dc286000d61d1130575c3e7dee88656f4b43fc2a5a05b97"
+                }
+            ]
+        },
+
+        {
+            "name": "gtkmm",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                   "type": "archive",
+                   "url": "https://ftp.acc.umu.se/pub/GNOME/sources/gtkmm/3.22/gtkmm-3.22.2.tar.xz",
+                   "sha256": "91afd98a31519536f5f397c2d79696e3d53143b80b75778521ca7b48cb280090"
+                }
+            ]
+        },
+
+        {
+            "name": "libxml++-2.6",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/libxml++2.6/2.40.1-2/libxml++2.6_2.40.1.orig.tar.xz",
+                    "sha256": "4ad4abdd3258874f61c2e2a41d08e9930677976d303653cd1670d3e9f35463e9"
+                }
+            ]
+        },
+
+        {
+            "name": "libpci",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/utils/pciutils/pciutils-3.6.2.tar.xz",
+                    "md5": "77963796d1be4f451b83e6da28ba4f82"
+                }
+            ],
+            "build-commands": [
+                "make PREFIX=/app SHAREDIR=/app/share/hwdata SHARED=yes",
+                "make PREFIX=/app SHAREDIR=/app/share/hwdata SHARED=yes install install-lib",
+                "chmod -v 755 /app/lib/libpci.so"
+            ]
+        },
+
+        {
+            "name": "adriconf",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/jlHertel/adriconf",
+                    "branch": "master"
```

Comment:
this needs to be a single released, reproducible commit or tarball to be on flathub (e.g. a branch or tag needs a pinned commit as well).